scripts/pylib/twister/twisterlib/environment.py

```diff
@@ -873,6 +873,18 @@ def parse_arguments(
         logger.error("west-flash requires device-testing to be enabled")
         sys.exit(1)
 
+    if options.device_serial_pty and options.device_serial_pty == "rtt":
+        if options.west_flash is None:
+            logger.error("--device-serial-pty rtt requires --west-flash")
```

Comment:
`--device-serial-pty` accept a script name, can be anything, here you assume that the script name (is this a script?) is always going to be rtt, or asking that if someone want to use this, they have to call their script rtt?

Reply:
No, in this case rtt is a keyword, and twister simply re-uses west's rtt integration.

Normally, the argument for device-serial-pty is a script through, so that hasn't changed.